### PR TITLE
GSoC2022: Add info to flyers and German translation of the flyer

### PR DESCRIPTION
**Pull request by Echolon**

### CODE REVIEW

File: content/community/gsoc-2022.de.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ Schauen dir auch unsere Medienkanäle an!
 - [Mastodon/Fediverse](https://fosstodon.org/@xmpp/)
 - [Youtube](https://www.youtube.com/c/XMPPStandardsFoundation)
 
+Darüber hinaus seid ihr eingeladen, mit unserem [Flyer vor Ort zu werben](https://github.com/xsf/xmpp.org/blob/master/static/images/promo/Flyer_XMPP_GSoC2022_DE.pdf), wie z.B. ein schwarzes Brett, für die bevorstehende Veranstaltung. Bitte seid so nett und fragt, ob ihr den Flyer aufhängen dürft.
```
Comment (wurstsalat3000) [inline content/community/gsoc-2022.de.md:37]: The link would have to be `/images/promo/Flyer_XMPP_GSoC2022_DE.pdf`

Same goes for the EN variant

Comment (Echolon) [reply content/community/gsoc-2022.de.md:37]: Shiat...

**Comment by Echolon** [comment]
# :heart:

**Comment by Echolon** [comment]
Hi @MattJ or @Zash

I would like to request a new website deployment if you find some time soon :pray:

**Pull request merged**